packages/node/src/integrations/hapi/types.ts

```diff
@@ -0,0 +1,270 @@
+/* eslint-disable @typescript-eslint/no-misused-new */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/unified-signatures */
+/* eslint-disable @typescript-eslint/no-empty-interface */
+/* eslint-disable @typescript-eslint/no-namespace */
+
+// Vendored and simplified from:
```

Comment:
Could we possibly add the versions here we vendored from, may make future updating easier? 😅